From 6c0412de4d3818eeee9b9ec3eb41d3695d2b498c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:06:55 -0400
Subject: [usb] uas: remove BROKEN

Message-id: <1410887463-20674-344-git-send-email-dzickus@redhat.com>
Patchwork-id: 95426
O-Subject: [RHEL7 PATCH 343/591] uas: remove BROKEN
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit dc88608dba784f902b3127fd68d0c4f92a532cd0
Author: Gerd Hoffmann <kraxel@redhat.com>
Date:   Fri Sep 13 13:27:15 2013 +0200

    uas: remove BROKEN

    xhci streams support is fixed, unblock usb attached scsi.

    Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/storage/Kconfig b/drivers/usb/storage/Kconfig
index 5939570..83e7d77 100644
--- a/drivers/usb/storage/Kconfig
+++ b/drivers/usb/storage/Kconfig
@@ -202,7 +202,7 @@ config USB_STORAGE_ENE_UB6250
 
 config USB_UAS
  tristate "USB Attached SCSI"
- depends on SCSI && USB_STORAGE && BROKEN
+ depends on SCSI && USB_STORAGE
  help
    The USB Attached SCSI protocol is supported by some USB
    storage devices.  It permits higher performance by supporting
-- 
1.7.1